Cargo.toml

```diff
@@ -13,6 +13,8 @@ members = [
 ]
 
 [workspace.dependencies]
+# note: bytemuck version must be available in all deployment environments
```

Comment:
Let's go one step more detailed and say it should be the floor of google3 and Chrome?

Reply:
sg, done